2-structured-data/config.js

```diff
@@ -51,7 +51,7 @@ nconf
   });
 
 // Check for required settings
-checkConfig('GCLOUD_PROJECT');
+checkConfig('GOOGLE_CLOUD_PROJECT');
```

Comment:
Why is the project ID a required config? 

The code is supposed to take it from the metadata server or from the GOOGLE_APPLICATION_CREDENTIAL context.